libnmstate/prettystate.py

```diff
@@ -56,11 +56,18 @@ def format_desired_current_state_diff(desired_state, current_state):
 class PrettyState(object):
     def __init__(self, state):
         yaml.add_representer(OrderedDict, represent_ordereddict)
+
+        # unicode does not exist in Python 3. Ignore it there.
+        try:
+            yaml.add_representer(unicode, represent_unicode)
+        except NameError:
+            pass
         self.state = order_state(deepcopy(state))
 
     @property
     def yaml(self):
-        return yaml.dump(self.state, default_flow_style=False)
+        return yaml.dump(self.state, default_flow_style=False,
+                         explicit_start=True)
```

Comment:
This is the start marker? Why is it needed/useful?

Reply:
yamllint warns if it is missing.